version.php

```diff
@@ -30,4 +30,4 @@
 $plugin->requires  = 2013111811;
 $plugin->component = 'local_aws';
 $plugin->maturity  = MATURITY_STABLE;
-$plugin->supported = [26, 311]; // A range of branch numbers of supported moodle versions.
+$plugin->supported = [26, 401]; // A range of branch numbers of supported moodle versions.
```

Comment:
@marxjohnson this is good to go, can you please bump the version and then merge away